frontend/device/sony-prstux/device.lua

```diff
@@ -15,6 +15,7 @@ local SonyPRSTUX = Generic:new{
     canReboot = yes,
     canPowerOff = yes,
     usbPluggedIn = false,
+    home_dir = nil,
```

Comment:
inherited from generic device, no need to override.

Reply:
I wanted to make it explicit that this is broken.